HW2Question1.cpp

```diff
@@ -0,0 +1,77 @@
+//
+//  HW2Question1.cpp
+//  Codu
+//
+//  Created by Farida Eid on 5/27/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+struct treeNode
+{
+    int value;
+    treeNode *left;
+    treeNode *right;
+    
+    treeNode(int v):value(v), left(NULL), right(NULL){}
+
+};
+
+bool printAncestors(treeNode *head, int key, vector<int>& answer)
```

Comment:
Also, it's usually better to use pointers when you modify the parameter, this makes it explicit to the caller, consider:
printAncestors(head, key, answer)
vs
printAncestors(head, key, &answer)